### ISSUE TITLE
Are there any plans to support Android in the near future?

### ISSUE THREAD

**Issue by lumotime** [hora-search/hora#15]
Are there any plans to support Android in the near future?

**Comment by salamer** [comment]
yes, we already have a JNI library (https://github.com/hora-search/hora-java), and we would support Android platform very soon.
